client/packages/coldchain/src/Equipment/ListView/ListView.tsx

```diff
@@ -105,6 +105,10 @@ const AssetListComponent: FC = () => {
       Cell: DotCell,
       sortable: false,
     },
+    {
+      key: 'installationDate',
+      label: 'label.installation-date',
+    },
```

Comment:
I'm just noticing the positioning isn't on the far right. But maybe notes are even less important?

Over to you but just flagging

Reply:
Note usually go on the far right so it's kep there for consistency...